Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -318,8 +324,12 @@ String CanvasRenderingContext2DBase::State::fontString() const
             family = family.substring(8);
 
         auto separator = i ? ", " : " ";
-        auto quote = family.contains(' ') ? "\"" : "";
-        serializedFont.append(separator, quote, family, quote);
+        if (family.containsOnly<isASCIIAlphaOrHyphen>())
```

Comment:
This check looks really close to what `serializeFontFamily` is doing in CSSMarkup, maybe we can reuse?

Reply:
Ooh, perfect, thanks!